spec/integration/action_spec.rb

```diff
@@ -1,31 +1,150 @@
 require 'verbalize/action'
 
 describe Verbalize::Action do
-  describe '.verbalize' do
+  let(:a) { double(divide_by: 20) }
+
+  let(:simple_action) do
+    Class.new do
+      include Verbalize::Action
+
+      input :a, :b, :should_fail
```

Comment:
I know this is just an example for a test, but passing in this *should_fail* is a bit confusing. Especially since the failure message is 'you can't divide by zero' but you're not dividing by zero.